### PR TITLE
solution

### PR DESCRIPTION
**Pull request by igor-kashchenko**
[DEMO LINK](https://igor-kashchenko.github.io/react_list-of-goods/)

### CODE REVIEW

**Review by Anastasiia-Khudych — APPROVED** [review]
Good job!

File: src/App.tsx [hunk 1]
```diff
@@ -26,70 +27,131 @@ type ReorderOptions = {
...
+  };
+
+  render() {
+    const {
+      sortType,
+      isReversed,
+    } = this.state;
+
+    const visibleGoods
+    = getReorderedGoods(goodsFromServer, { sortType, isReversed });
```
Comment (Anna-Tkachenko) [inline src/App.tsx:100]:
```suggestion
    const visibleGoods = getReorderedGoods(
      goodsFromServer,
      { sortType, isReversed },
    );
```

File: src/App.tsx [hunk 2]
```diff
@@ -26,70 +27,131 @@ type ReorderOptions = {
...
+          <button
+            type="button"
+            className={classNames('button is-warning', {
+              'is-light': !isReversed,
+            })}
+            onClick={this.reverseGoods}
+          >
+            Reverse
+          </button>
+          {(isReversed || sortType !== SortType.NONE)
```
Comment (Anna-Tkachenko) [inline src/App.tsx:134]: create separate boolean variable before `return` statement and store there the result of this check
in this way your code will be clearer
don't combine several checks directly in `jsx`

**Review by Anna-Tkachenko — CHANGES_REQUESTED** [review]

**Review by Anna-Tkachenko — APPROVED** [review]
⭐️

**Pull request closed without merging**